src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -343,7 +335,7 @@ void ParticipantConfiguration::xmlTagCallback(
     std::string        meshName = tag.getStringAttributeValue(ATTR_MESH);
     mesh::PtrMesh      mesh     = _meshConfig->getMesh(meshName);
     PRECICE_CHECK(mesh,
-                  "Participant \"{}\" has to use mesh \"{}\" in order to read data from it. Please add a use-mesh node with name=\"{}\".",
+                  "Participant \"{}\" has to receive mesh \"{}\" in order to read data from it. Please add a receive-mesh node with name=\"{}\".",
```

Comment:
Both is allowed
* provided meshes for data that is mapped to a local mesh
* received meshes for direct access

Reply:
Correct. However, most of our users don't use direct access, I would propose to make the message a but more exhaustive to distinguish both use cases. For the next release we also want to make the direct access non-experimental so that it would make sense to have it in the error message by default.

Reply:
The check is incorrect. We need to rethink this.